app/src/main/java/org/simple/clinic/bp/entry/BpValidator.kt

```diff
@@ -0,0 +1,52 @@
+package org.simple.clinic.bp.entry
+
+import org.simple.clinic.bp.entry.BpValidator.Validation.ErrorDiastolicEmpty
+import org.simple.clinic.bp.entry.BpValidator.Validation.ErrorDiastolicTooHigh
+import org.simple.clinic.bp.entry.BpValidator.Validation.ErrorDiastolicTooLow
+import org.simple.clinic.bp.entry.BpValidator.Validation.ErrorSystolicEmpty
+import org.simple.clinic.bp.entry.BpValidator.Validation.ErrorSystolicLessThanDiastolic
+import org.simple.clinic.bp.entry.BpValidator.Validation.ErrorSystolicTooHigh
+import org.simple.clinic.bp.entry.BpValidator.Validation.ErrorSystolicTooLow
+import org.simple.clinic.bp.entry.BpValidator.Validation.Success
+import javax.inject.Inject
+
+class BpValidator @Inject constructor() {
+
+  // TODO: Rename to "Result".
```

Comment:
Do you want to complete these `TODO`s? They seem to be trivial.

Reply:
I'll do this in my next PR